### ISSUE TITLE
[expression:1305]FUNCTION LowerUTF8 does not exist

### ISSUE THREAD

**Issue by qw4990** [pingcap/tidb#33145]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->
```
mysql> SELECT QUERY_SAMPLE_TEXT FROM information_schema.cluster_statements_summary_history WHERE lower(QUERY_SAMPLE_TEXT) LIKE '%select%';
ERROR 1105 (HY000): other error: [expression:1305]FUNCTION LowerUTF8 does not exist
```

### 2. What did you expect to see? (Required)
no error

### 3. What did you see instead (Required)
```
mysql> SELECT QUERY_SAMPLE_TEXT FROM information_schema.cluster_statements_summary_history WHERE lower(QUERY_SAMPLE_TEXT) LIKE '%select%';
ERROR 1105 (HY000): other error: [expression:1305]FUNCTION LowerUTF8 does not exist
```

### 4. What is your TiDB version? (Required)

<!-- Paste the output of SELECT tidb_version() -->

```
mysql> select tidb_version();
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| tidb_version()                                                                                                                                                                                                                                                                                     |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Release Version: v5.3.0
Edition: Community
Git Commit Hash: 4a1b2e9fe5b5afb1068c56de47adb07098d768d6
Git Branch: HEAD
UTC Build Time: 2022-03-16 08:03:24
GoVersion: go1.16.3
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
1 row in set (0.00 sec)
```

**Comment by zanmato1984** [comment]
Reproduced on 5.3.0 and 5.4.0. Not seen on 6.0.0. Adjusting labels.

**Comment by zanmato1984** [comment]
Already fixed in https://github.com/pingcap/tidb/pull/32553

**Comment by zanmato1984** [comment]
Fixed since 6.0. Closing.
